README.md

```diff
@@ -162,6 +162,11 @@ client library supporting this) any pushes with timestamps will be rejected.
 If you think you need to push a timestamp, please see [When To Use The
 Pushgateway](https://prometheus.io/docs/practices/pushing/).
 
+In order to make it easier to alert on pushers that have not run recently, the
+push gateway will add in a metric `push_time_seconds` with the unix timestamp
```

Comment:
And now that we are concerned with spelling nits: I'd capitalize Unix. Cf. https://en.wikipedia.org/wiki/Unix_time .